docs/Modern-Introduction.md

```diff
@@ -41,9 +41,9 @@ Run the Relay Compiler after making changes to any GraphQL in your Relay applica
 "relay": "relay-compiler --src ./src --schema path/schema.graphql"
 ```
 
+If you are using different extension like jsx for your javascript files add `--extensions jsx` parameter at the end of a compiler command.
```

Comment:
I don’t know if what I’m suggesting is the previous version you had that @nodkz asked to change, so putting it out there just in case 😄 

I think that adding the option to the example command might be more helpful instead of adding another sentence. E.g.

```js
"relay": "relay-compiler --src ./src --schema path/schema.graphql --extensions js jsx"
```